DEVELOPERS.md

```diff
@@ -10,29 +10,33 @@ Due to browser security restrictions, we can't communicate with dapps running on
 
 ### :partly_sunny: Web3 - Ethereum Browser Environment Check
 
-MetaMask injects `web3` object and convenience Web3.js library into the javascript context.
-Look for this before using your fallback strategy (local node / hosted node + in-dapp id mgmt / read-only / fail).
-You can use the injected Web3.js directly, but the best practice is to explicitly bundle **the version of web3.js you used during development**.
-
-Note that the environmental `web3` check is wrapped in a `window.addEventListener('load', ...)` handler. This approach avoids race conditions with web3 injection timing.
+The Ethereum provider injected by MetaMask and other dapp browsers is available at `window.ethereum`. Before reading user accounts or initiating RPC method calls that require user accounts, such as `eth_sendTransaction`, dapps must now request access to user accounts by calling a new method on the provider: `ethereum.enable()`. This method returns a Promise that’s either resolved with user accounts after user approval, or rejected with an Error after user rejection.
```

Comment:
Given that a deeper explanation of ethereum providers has been submitted in #117, it may be worth looking at that and maybe replacing this with that.

Reply:
According to the latest version of EIP1102 https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1102.md
The `ethereum.enable()` has **DEPRECATED**,  and use new rpc method `eth_requestAccounts` now.

And about `send` method like:
```javascript
 ethereum.send('eth_sendTransaction', { from: accounts[0], /* ... */ })
```
The params of `send` method is not compatible with `httpProvider` of Web3.

What's the next plan of metaMask ?